src/taipy/core/_repository/_v2/_abstract_repository.py

```diff
@@ -15,15 +15,12 @@
 from abc import abstractmethod
 from datetime import datetime, timedelta
 from enum import Enum
-from typing import Any, Dict, Generic, Iterable, List, Optional, TypeVar, Union
+from typing import Any, Dict, Generic, Iterable, List, Optional, Union
 
-Json = Union[dict, list, str, int, float, bool, None]
+from src.taipy.core.common.typing import Entity, Json, ModelType
```

Comment:
I believe it must be a relative path. otherwise it won't work on taipy/enterprise/rest repos.